### ISSUE TITLE
Import new CA into Trusted Certs

### ISSUE THREAD

**Issue by pabluk** [peopledoc/ansible-role-oracle-java#13]
It would be great to be able to be able to install new CA certificates using this role.

This is the command to execute to import a new CA:
```bash
keytool -import -noprompt -storepass xxxxx -trustcacerts \
 -file /usr/local/share/ca-certificates/my-ca.crt -alias my-ca \
 -keystore /usr/lib/jvm/java-8-oracle/jre/lib/security/cacerts
```

**Comment by pilou-** [comment]
It seems Ansible [java_cert](http://docs.ansible.com/ansible/2.4/java_cert_module.html) should be used instead. A role should not be used when a module could be used instead, of course if it makes sense this role could use the module.

**Comment by tyoras** [comment]
good idea @pabluk and I agree with you @pilou- if a module exist for this we should use it in this module

**Comment by pilou-** [comment]
Fixed by #15 .
